docs/v1/services/mysql/mysql.md

```diff
@@ -81,13 +81,21 @@ Here is a [sample YAML](/config/samples/azure_v1alpha1_mysqlserveradministrator.
 
 The MySQL user operator allows you to add a new user to an existing MySQL database. 
 
-Here is a [sample YAML](/config/samples/azure_v1alpha1_mysqluser.yaml) for MySQL user. 
+Here is a [sample YAML](/config/samples/azure_v1alpha2_mysqluser.yaml) for MySQL user. 
 
 The `resourceGroup` is the resource group of the MySQL server and MySQL database, provide the MySQL server name in `server` and MySQL database name in `dbName`. 
 
-The operator supports grant specified privileges using the concept of `roles`, and supports assigning one or more privileges from the list:
+The operator supports granting the user global privileges using the `roles` field, and supports assigning one or more privileges from the list:
 
-##### `SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, RELOAD, PROCESS, REFERENCES, INDEX, ALTER, SHOW DATABASES, CREATE TEMPORARY TABLES, LOCK TABLES, EXECUTE, REPLICATION SLAVE, REPLICATION CLIENT, CREATE VIEW, SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, CREATE USER, EVENT, TRIGGER`.
+##### `CREATE TABLESPACE, CREATE USER, FILE, PROCESS, RELOAD, REPLICATION CLIENT, REPLICATION SLAVE, SHOW DATABASES, SHUTDOWN, SUPER`
```

Comment:
Why do this as a heading?

```suggestion
* `CREATE TABLESPACE`
* `CREATE USER`
* `FILE`
* `PROCESS`
* `RELOAD`
* `REPLICATION CLIENT`
* `REPLICATION SLAVE`
* `SHOW DATABASES`
* `SHUTDOWN`
* `SUPER``
```

Same change for below.

Reply:
Yeah, fair point - I was just keeping the same format as previously without really thinking about it but a list is better.